### ISSUE TITLE
occ does not take unix path names into account when generating object files

### ISSUE THREAD

**Issue by GitMensch** [LADSoft/OrangeC#65]
Using latest CI generated binaries and

~~~
occ.exe -c -I.. -I../../orangec/include ../pdcurses/debug.c
~~~

produces `../pdcurses/debug.o` and then calls `olink` which errors with

> Error: Input file 'debug.o' does not exist.

--> the source file and the include files work as expected with the `/`
--> in this case the output file is placed where the source file is

Using

~~~
occ.exe -c -I.. -I../../orangec/include ..\pdcurses\debug.c
~~~

produces `debug.o` in current directory, `olink` therefore has no issues to find it.

Both should produce the same result.

**Comment by LADSoft** [comment]
I fixed it.

**Comment by GitMensch** [comment]
> I fixed it.

Just for reference as there was no commit referencing this (may be done by adding `#65` in the commit message): What commit (SHA256) contains the fix?

I suggest to keep this issue open until a minimal test is added (I don't know if you can assign a tag to this issue to show this).

**Comment by LADSoft** [comment]
yeah that was a problem.   I didn't know about the #65 thing, will start using it going forward.

I'm not sure how to reference a checkin here  so I'll just list the hashes.

Looks like I intended to fix it here: 19640808ac62f9f3adf3391fb13037d99fbda00c
 
but missed a file and it got checked in here: 14cb299b3f0b66035374a414e39f396220df55e6
